README.md

```diff
@@ -0,0 +1,112 @@
+
+Then, we can run the following command to build our app image - no additional steps needed! The required buildpacks for your app will be detected automatically.
+
+```
+pack build my-app-image --builder heroku/builder:22 --path node-js-getting-started
+```
+
+### Tutorial
+
+We also have a more in-depth tutorial available. It can be found in the [Heroku Buildpacks repository](https://github.com/heroku/buildpacks). 
+
+## Discussions
+
+Overarching discussions about the Heroku Buildpacks project happen on GitHub in the [discussions tab of this repository](https://github.com/heroku/buildpacks/discussions). Feel free to open a new discussion if you have any questions, feedback or other input - we want to hear from you!
+
+To view and report issues that are specific to a language ecosystem, see the issue tracker links in the ["Supported Languages" section](#supported-languages).  
+
+## Supported Languages
+
+Each supported ecosystem has its own Git repository where all related buildpacks are bundled. Language specific issues are tracked in their respective repositories.
+
+| Name    | Repository                             | Issue Tracker           | Buildpack Registry                                                                               |
+|---------|----------------------------------------|-------------------------|--------------------------------------------------------------------------------------------------|
+| Go      | [buildpacks-go][go-repository]         | [Issues][go-issues]     | [![Cloud Native Buildpacks Registry: heroku/go][go-registry-badge]][go-registry-url]             |
+| Java    | [buildpacks-jvm][jvm-repository]       | [Issues][jvm-issues]    | [![Cloud Native Buildpacks Registry: heroku/java][java-registry-badge]][java-registry-url]       |
+| Node.js | [buildpacks-nodejs][nodejs-repository] | [Issues][nodejs-issues] | [![Cloud Native Buildpacks Registry: heroku/nodejs][nodejs-registry-badge]][nodejs-registry-url] |
+| PHP     | [buildpacks-php][php-repository]       | [Issues][php-issues]    | [![Cloud Native Buildpacks Registry: heroku/php][php-registry-badge]][php-registry-url]          |
+| Python  | [buildpacks-python][python-repository] | [Issues][python-issues] | [![Cloud Native Buildpacks Registry: heroku/python][python-registry-badge]][python-registry-url] |
+| Ruby    | [buildpacks-ruby][ruby-repository]     | [Issues][ruby-issues]   | [![Cloud Native Buildpacks Registry: heroku/ruby][ruby-registry-badge]][ruby-registry-url]       |
+| Scala   | [buildpacks-jvm][jvm-repository]       | [Issues][jvm-issues]    | [![Cloud Native Buildpacks Registry: heroku/scala][scala-registry-badge]][scala-registry-url]    |
+
+## Associated Projects
```

Comment:
Should we add the procfile CNB, builder and base images repos to the "associated projects" list?